### ISSUE TITLE
We should have a way to check the current app version so we can ask the user to upgrade

### ISSUE THREAD

**Issue by perpetuatingcuriosity** [creativecommons/list#299]
@mattl Just adding this from our Monday chat. You were thinking there’s another way to do this other than an endpoint?

If it’s an issue of different versions of the app vs. webapp versions, maybe we can have an endpoint that lists all current builds?

e.g.: GET /api/info

```
[{
"api_version": "1.0",
"app_version": "3.0.1",
"webapp_version": "0.5",
}]
```

Whatever method we use, do you think we get can get this working for our API day 3 this month (4.0 internal release day)?

**Comment by mattl** [comment]
Yeah, I think there's a different way to do it, and one that shouldn't need an API call. 

I imagine that any List client can talk to any List server, providing an API level is supported. For this, I would like to make a simple text file available at the root of the server. I'll think about how we can do this.

**Comment by mattl** [comment]
At some point, the API will have an API key. I guess we could exclude this from the API-key requirement.

**Comment by perpetuatingcuriosity** [comment]
@mattl: Not sure I understand what you mean by “exclude this from the API-key requirement”? How does the api key affect the app version?

Can you provide this file with the app version before I start implementing this?

**Comment by mattl** [comment]
No, we should make it an API function, we should just be clear that this function doesn't require any API key.

**Comment by perpetuatingcuriosity** [comment]
Kk, let me know when the endpoint is working so I can start implementing it in the app.

**Comment by mattl** [comment]
I'll let you work on the end point, like with anonymous users. I have less than 0 time anytime soon for this, but we should start blocking older clients soon.

**Comment by perpetuatingcuriosity** [comment]
I can cover any android implementation in this issue. Requests to the api and UI components.
